package.json

```diff
@@ -64,6 +64,8 @@
     "node": ">=4.5"
   },
   "devDependencies": {
+    "@types/node": "^8.5.8",
```

Comment:
Ideally we could keep these to prevent issues, but if there's a solid reason for moving then 👍

Reply:
I think the issue is that Fastify works with Node 4, 6, 8 and 9. Fixing `@types/node` to a specific version is very confusing for a library.